server/graphql/general/src/queries/names.rs

```diff
@@ -44,12 +92,24 @@ pub struct NameFilterInput {
     pub is_store: Option<bool>,
     /// Code of the store if store is linked to name
     pub store_code: Option<SimpleStringFilterInput>,
-    /// Visibility in current store (based on store_id parameter and existance of name_store_join record)
+    /// Visibility in current store (based on store_id parameter and existence of name_store_join record)
     pub is_visible: Option<bool>,
     /// Show system names (defaults to false)
     /// System names don't have name_store_join thus if queried with true filter, is_visible filter should also be true or null
     /// if is_visible is set to true and is_system_name is also true no system names will be returned
     pub is_system_name: Option<bool>,
+    /// Filter by name type
+    pub r#type: Option<EqualFilterNameTypeInput>,
```

Comment:
I thought patients is it's own type and has it's own filter ?

Reply:
We've seperated filer for enums before (for statuses of shipments etc..), I think this should be a seperate filter